lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM.cpp

```diff
@@ -637,7 +637,13 @@ struct ConvertLayoutOpConversion
           processReplicaForMMAV1(loc, rewriter, /*stNotRd*/ true, srcTy,
                                  multiDimRepId, inVec, paddedRepShape, outOrd,
                                  vals, smemBase, shape);
-        else
+        else if (isStMatrixCompatible(srcTy) && accumNumReplicates == 1 &&
```

Comment:
`numRepliates` in here is calculated based on `shapePerWarp = [16, 8]`, but stmatrix does [16, 16], maybe a bug ?

Reply:
numRepliates is to handle multiple CTAs right? I'm skipping this case as this is not really enabled right now.